stock_request_picking_type/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Open Source Integrators
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Stock Request Picking Type",
+    "summary": "Add Stock Requests to the Inventory App",
+    "version": "13.0.1.0.0",
+    "license": "LGPL-3",
+    "website": "https://github.com/stock-logistics-warehouse",
+    "author": "Open Source Integrators, " "Odoo Community Association (OCA)",
+    "category": "Warehouse",
+    "depends": ["stock_request_submit"],
```

Comment:
Why depending on this module (apart for the search view - which IMHO should be defined in the base module) ?

IMHO, the subimtted state should be included in base module. @max3903 Any advice on this ?

Reply:
Agreed, stock_request_submit should be merged into stock_request. Its only function is to add the 'submitted' state to stock.request.order. This view should be merged with the searchview added in #903 .

Reply:
@ps-tubtim or @RLeeOSI would you improve this one.

Reply:
I changed this code from `"depends": ["stock_request_submit"],` to `"depends": ["stock_request"],`.